unime/src/routes/(app)/me/settings/app/+page.svelte

```diff
@@ -2,11 +2,11 @@
   import { goto } from '$app/navigation';
 
   import { locales } from '$lib/app/locales';
-  import { dispatch } from '$lib/dispatcher';
   import LL from '$src/i18n/i18n-svelte';
   import SettingsEntry from '$src/lib/app/settings/SettingsEntry.svelte';
   import Switch from '$src/lib/components/atoms/Switch.svelte';
   import TopNavBar from '$src/lib/components/molecules/navigation/TopNavBar.svelte';
+  import { dispatch } from '$src/lib/dispatcher';
```

Comment:
Did you change that on purpose from `$lib` to `$src/lib`? It doesn't really make a difference except that it's shorter and as you can see we're not consistent at all, I'm just curious if there was an intention behind it. 🙂

Reply:
Maybe because I removed it accidentally and readded it again through editor. Changed it back now